lib/rubocop/cop/style/access_modifier_declarations.rb

```diff
@@ -65,6 +65,9 @@ def on_send(node)
           return unless node.access_modifier?
           return if node.parent.pair_type?
 
+          argument = node.children.last
```

Comment:
I wonder if we can simplify this somehow. Generally I've never been fond of getting all the children in this fashion. I'd rather go with simple destructuring of the node.